### PR TITLE
v1.12.0: some fixes to CHANGELOG

### PR DESCRIPTION
**Pull request by tiborvass**
Updates to CHANGELOG for rc2

### CODE REVIEW

**Comment by thaJeztah** [comment]
Didn't find https://github.com/docker/docker/pull/19367 in there

**Comment by thaJeztah** [comment]
These as well:

- https://github.com/docker/docker/pull/20410
- https://github.com/docker/docker/pull/20835
- https://github.com/docker/docker/pull/20853
- https://github.com/docker/docker/pull/21006
- https://github.com/docker/docker/pull/21361
- https://github.com/docker/docker/pull/21495
- https://github.com/docker/docker/pull/21556
- https://github.com/docker/docker/pull/21641
- https://github.com/docker/docker/pull/21724
- https://github.com/docker/docker/pull/21773

**Comment by thaJeztah** [comment]
- https://github.com/docker/docker/pull/21844
- https://github.com/docker/docker/pull/21901
- https://github.com/docker/docker/pull/21904
- https://github.com/docker/docker/pull/21945
- https://github.com/docker/docker/pull/22000 (although the env-var is gone, so may not be needed)
- https://github.com/docker/docker/pull/22094
- https://github.com/docker/docker/pull/22138
- https://github.com/docker/docker/pull/22344
- https://github.com/docker/docker/pull/22408
- https://github.com/docker/docker/pull/22448
- https://github.com/docker/docker/pull/22508

**Comment by thaJeztah** [comment]
- https://github.com/docker/docker/pull/22932

**Comment by thaJeztah** [comment]
@tiborvass don't know if you included this one, but this was in the TP5 milestone for some reason https://github.com/docker/docker/pull/20639

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,9 +15,12 @@ be found.
 + Add support for comments in `.dockerignore` [#23111](https://github.com/docker/docker/pull/23111)
 * Support for UTF-8 in Dockerfiles [#23372](https://github.com/docker/docker/pull/23372)
 * Skip UTF-8 BOM bytes from `Dockerfile` and `.dockerignore` if exist [#23234](https://github.com/docker/docker/pull/23234)
+* windows: support for `ARG` to match Linux [#22508](https://github.com/docker/docker/pull/22508)
```
Comment (mlaventure) [inline CHANGELOG.md]: Should we have the same style as the other entries? e.g.: "Support for `ARG` is now also available in Windows`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,28 +33,45 @@ be found.
 
 + Syslog logging driver now supports DGRAM sockets [#21613](https://github.com/docker/docker/pull/21613)
 + Add `--details` option to `docker logs` to also display log tags [#21889](https://github.com/docker/docker/pull/21889)
++ Enable syslog logger to have access to env and labels [#21724](https://github.com/docker/docker/pull/21724)
++ Additional syslog-format option to allow microsecond resolution in syslog timestamp [#21844](https://github.com/docker/docker/pull/21844)
```
Comment (mlaventure) [inline CHANGELOG.md]: **An** additional syslog-format option **(`rfc5424micro`) has been added** to allow microsecond resolution in syslog timestamp ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -90,8 +115,11 @@ be found.
 ### Volume
 
 + Add support for local and global volume scopes (analogous to network scopes) [#22077](https://github.com/docker/docker/pull/22077)
++ Allow volume drivers to provide a `Status` field [#21006](https://github.com/docker/docker/pull/21006)
++ Add name/driver filter support for volume [#21361](https://github.com/docker/docker/pull/21361)
 * Mount/Unmount operations now receives an opaque ID to allow volume drivers to differentiate between two callers [#21015](https://github.com/docker/docker/pull/21015)
 - Fix issue preventing to remove a volume in a corner case [#22103](https://github.com/docker/docker/pull/22103)
+- windows: Enable auto-creation of host-path to match Linux [#22094](https://github.com/docker/docker/pull/22094)
```
Comment (mlaventure) [inline CHANGELOG.md]: I think Windows is supposed to be capitalized

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,9 +15,12 @@ be found.
 + Add support for comments in `.dockerignore` [#23111](https://github.com/docker/docker/pull/23111)
 * Support for UTF-8 in Dockerfiles [#23372](https://github.com/docker/docker/pull/23372)
 * Skip UTF-8 BOM bytes from `Dockerfile` and `.dockerignore` if exist [#23234](https://github.com/docker/docker/pull/23234)
+* windows: support for `ARG` to match Linux [#22508](https://github.com/docker/docker/pull/22508)
+- Fix error message when building using a daemon with the bridge network disabled [#22932](https://github.com/docker/docker/pull/22932)
 
 ### Contrib
 
+* Centos 7 and Oracle Linux 7 now include libseccomp 2.2.1 [#22344](https://github.com/docker/docker/pull/22344)
```
Comment (thaJeztah) [inline CHANGELOG.md]: "Enable seccomp support for CentOS 7 and Oracle Linux 7" ?

**Comment by thaJeztah** [comment]
LGTM

**Comment by mlaventure** [comment]
LGTM (IANAM)

**Comment by tiborvass** [comment]
SELF LGTM

**Comment by tiborvass** [comment]
Changelog merged to master in #25238

**Pull request merged**